integration-tests/http-backend-request-error-handling.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { HttpBackend } from '@taquito/http-utils';
+
+describe('HttpBackend request', () => {
+    it('will fail with http request url and request method', async (done) => {
+        try {
+            const http: HttpBackend = new HttpBackend(1);
+            await http.createRequest<string>({
+              method: 'GET',
+              url: 'https://mainnet.api.tez.ie/chains/main/blocks/head/hash'
```

Comment:
Could you also add an expect statement for the message to make sure it renders properly?

Reply:
yes, added expect statement with message and pushed another commit